mlir/include/mlir/Dialect/MIOpen/utility/common.hpp

```diff
@@ -0,0 +1,8 @@
+#ifndef COMMON_H
```

Comment:
please rename the file to `common.h`.

same with `math.hpp`. please help rename it to `math.h` to cope with LLVM / MLIR naming convention.

Reply:
done